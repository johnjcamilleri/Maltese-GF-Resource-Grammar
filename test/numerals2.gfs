-- Treebank numerals test (>1000)
-- Run using the following command:
-- ./_gf < test/numerals2.gfs > test/numerals2.out

se utf8
i NumeralMlt.gf
rf -lines -tree -file=test/numerals2.trees | l -table
