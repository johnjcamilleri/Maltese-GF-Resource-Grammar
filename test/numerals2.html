<html>
<head>
	<meta http-equiv="Content-type" content="text/html;charset=UTF-8">
	<style type="text/css">
		body { font: 16px/1.5 serif; }
		div { margin-left: 2em; }
		#text-output { margin:0; color:#999; }
		#text-output b { display: inline-block; width: 10em; font-weight:normal; }
		#text-output em { font-style: italic; color:#000; }
		#text-output em.correct, .correct { color:green; }
		#text-output em.incorrect, .incorrect { color:red; }
		#text-output em.gold, .gold { background-color:gold; padding:0 5px; }

		table { border-collapse:collapse; border-spacing:0; }
		tr:nth-child(odd) { background-color:#eee; }
		tr.separator { background-color:#000; font-size:2px; }
		td { padding:0.2em 1em; border-width:1px 0; border-style:solid; border-color:#ccc; }
		td:last-child { font-style:italic; }

	</style>
</head>
<body>

<h1>Tree linearisations for: numerals2</h1>
<p><em>Generated Mon, 20 Feb 2012 10:07:14 +0100</em></p>
<p><span class="correct">140 correct</span>, <span class="incorrect">16 incorrect</span></p>
<hr/>

<div id="text-output">
<div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>elf</em></div><div><b>NumAdjectival</b><em>elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>l- elf</em></div><div><b>NumAdjectival</b><em>l- elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>elfejn</em></div><div><b>NumAdjectival</b><em>elfejn</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>l- elfejn</em></div><div><b>NumAdjectival</b><em>l- elfejn</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>tlett elef</em></div><div><b>NumAdjectival</b><em>tlett elef</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>it- tlett elef</em></div><div><b>NumAdjectival</b><em>it- tlett elef</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>disat elef</em></div><div><b>NumAdjectival</b><em>disat elef</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>id- disat elef</em></div><div><b>NumAdjectival</b><em>id- disat elef</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>għaxart elef</em></div><div><b>NumAdjectival</b><em>għaxart elef</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>l- għaxart elef</em></div><div><b>NumAdjectival</b><em>l- għaxart elef</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>ħdax-il elf</em></div><div><b>NumAdjectival</b><em>ħdax-il elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- ħdax-il elf</em></div><div><b>NumAdjectival</b><em>il- ħdax-il elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>tnax-il elf</em></div><div><b>NumAdjectival</b><em>tnax-il elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>it- tnax-il elf</em></div><div><b>NumAdjectival</b><em>it- tnax-il elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>tlettax-il elf</em></div><div><b>NumAdjectival</b><em>tlettax-il elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>it- tlettax-il elf</em></div><div><b>NumAdjectival</b><em>it- tlettax-il elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>dsatax-il elf</em></div><div><b>NumAdjectival</b><em>dsatax-il elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>id- dsatax-il elf</em></div><div><b>NumAdjectival</b><em>id- dsatax-il elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>għoxrin elf</em></div><div><b>NumAdjectival</b><em>għoxrin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>l- għoxrin elf</em></div><div><b>NumAdjectival</b><em>l- għoxrin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>wieħed u għoxrin elf</em></div><div><b>NumAdjectival</b><em>wieħed u għoxrin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- wieħed u għoxrin elf</em></div><div><b>NumAdjectival</b><em>il- wieħed u għoxrin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>tnejn u għoxrin elf</em></div><div><b>NumAdjectival</b><em>tnejn u għoxrin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>it- tnejn u għoxrin elf</em></div><div><b>NumAdjectival</b><em>it- tnejn u għoxrin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>tletin elf</em></div><div><b>NumAdjectival</b><em>tletin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>it- tletin elf</em></div><div><b>NumAdjectival</b><em>it- tletin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>wieħed u tletin elf</em></div><div><b>NumAdjectival</b><em>wieħed u tletin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- wieħed u tletin elf</em></div><div><b>NumAdjectival</b><em>il- wieħed u tletin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>tnejn u tletin elf</em></div><div><b>NumAdjectival</b><em>tnejn u tletin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>it- tnejn u tletin elf</em></div><div><b>NumAdjectival</b><em>it- tnejn u tletin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>disgħa u disgħin elf</em></div><div><b>NumAdjectival</b><em>disgħa u disgħin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>id- disgħa u disgħin elf</em></div><div><b>NumAdjectival</b><em>id- disgħa u disgħin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mitt elf</em></div><div><b>NumAdjectival</b><em>mitt elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mitt elf</em></div><div><b>NumAdjectival</b><em>il- mitt elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mija u wieħed elf</em></div><div><b>NumAdjectival</b><em>mija u wieħed elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mija u wieħed elf</em></div><div><b>NumAdjectival</b><em>il- mija u wieħed elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mija u sitt elef</em></div><div><b>NumAdjectival</b><em>mija u sitt elef</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mija u sitt elef</em></div><div><b>NumAdjectival</b><em>il- mija u sitt elef</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mija u għaxart elef</em></div><div><b>NumAdjectival</b><em>mija u għaxart elef</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mija u għaxart elef</em></div><div><b>NumAdjectival</b><em>il- mija u għaxart elef</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mija u ħdax-il elf</em></div><div><b>NumAdjectival</b><em>mija u ħdax-il elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mija u ħdax-il elf</em></div><div><b>NumAdjectival</b><em>il- mija u ħdax-il elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mija u tletin elf</em></div><div><b>NumAdjectival</b><em>mija u tletin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mija u tletin elf</em></div><div><b>NumAdjectival</b><em>il- mija u tletin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mija u wieħed u erbgħin elf</em></div><div><b>NumAdjectival</b><em>mija u wieħed u erbgħin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mija u wieħed u erbgħin elf</em></div><div><b>NumAdjectival</b><em>il- mija u wieħed u erbgħin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mija u sittin elf</em></div><div><b>NumAdjectival</b><em>mija u sittin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mija u sittin elf</em></div><div><b>NumAdjectival</b><em>il- mija u sittin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mitejn elf</em></div><div><b>NumAdjectival</b><em>mitejn elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mitejn elf</em></div><div><b>NumAdjectival</b><em>il- mitejn elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>erba' mija u sittax-il elf</em></div><div><b>NumAdjectival</b><em>erba' mija u sittax-il elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>l- erba' mija u sittax-il elf</em></div><div><b>NumAdjectival</b><em>l- erba' mija u sittax-il elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>ħdax-il elf u tnax</em></div><div><b>NumAdjectival</b><em>ħdax-il elf u tnax-il</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- ħdax-il elf u tnax</em></div><div><b>NumAdjectival</b><em>il- ħdax-il elf u tnax-il</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>erbgħin elf</em></div><div><b>NumAdjectival</b><em>erbgħin elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>l- erbgħin elf</em></div><div><b>NumAdjectival</b><em>l- erbgħin elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>wieħed u disgħin elf u seba' mija</em></div><div><b>NumAdjectival</b><em>wieħed u disgħin elf u seba' mitt</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- wieħed u disgħin elf u seba' mija</em></div><div><b>NumAdjectival</b><em>il- wieħed u disgħin elf u seba' mitt</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mitt elf u mija</em></div><div><b>NumAdjectival</b><em>mitt elf u mitt</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mitt elf u mija</em></div><div><b>NumAdjectival</b><em>il- mitt elf u mitt</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mitt elf u wieħed u tletin</em></div><div><b>NumAdjectival</b><em>mitt elf u wieħed u tletin</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mitt elf u wieħed u tletin</em></div><div><b>NumAdjectival</b><em>il- mitt elf u wieħed u tletin</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mija u wieħed u għoxrin elf u mija</em></div><div><b>NumAdjectival</b><em>mija u wieħed u għoxrin elf u mitt</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mija u wieħed u għoxrin elf u mija</em></div><div><b>NumAdjectival</b><em>il- mija u wieħed u għoxrin elf u mitt</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>mitejn elf u erba' mija u għaxra</em></div><div><b>NumAdjectival</b><em>mitejn elf u erba' mija u għaxar</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>il- mitejn elf u erba' mija u għaxra</em></div><div><b>NumAdjectival</b><em>il- mitejn elf u erba' mija u għaxar</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em class='incorrect'>sitt elf</em> &rarr; <em class='gold'>sitt mitt elf</em></div><div><b>NumAdjectival</b><em class='incorrect'>sitt elf</em> &rarr; <em class='gold'>sitt mitt elf</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em class='incorrect'>is- sitt elf</em> &rarr; <em class='gold'>is- sitt mitt elf</em></div><div><b>NumAdjectival</b><em class='incorrect'>is- sitt elf</em> &rarr; <em class='gold'>is- sitt mitt elf</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>sitt mija u għaxart elef u għaxra</em></div><div><b>NumAdjectival</b><em>sitt mija u għaxart elef u għaxar</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>is- sitt mija u għaxart elef u għaxra</em></div><div><b>NumAdjectival</b><em>is- sitt mija u għaxart elef u għaxar</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em class='incorrect'>sebat elf u sitt mija u għaxra</em> &rarr; <em class='gold'>sebat elef u sitt mija u għaxra</em></div><div><b>NumAdjectival</b><em class='incorrect'>sebat elf u sitt mija u għaxar</em> &rarr; <em class='gold'>sebat elef u sitt mija u għaxar</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em class='incorrect'>is- sebat elf u sitt mija u għaxra</em> &rarr; <em class='gold'>is- sebat elef u sitt mija u għaxra</em></div><div><b>NumAdjectival</b><em class='incorrect'>is- sebat elf u sitt mija u għaxar</em> &rarr; <em class='gold'>is- sebat elef u sitt mija u għaxar</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em>tmin mija u ħamsin elf u sbatax</em></div><div><b>NumAdjectival</b><em>tmin mija u ħamsin elf u sbatax-il</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em>it- tmin mija u ħamsin elf u sbatax</em></div><div><b>NumAdjectival</b><em>it- tmin mija u ħamsin elf u sbatax-il</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em class='incorrect'>disat elf u mija u għaxra</em> &rarr; <em class='gold'>disa' mitt elf u mija u għaxra</em></div><div><b>NumAdjectival</b><em class='incorrect'>disat elf u mija u għaxar</em> &rarr; <em class='gold'>disa' mitt elf u mija u għaxar</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em class='incorrect'>id- disat elf u mija u għaxra</em> &rarr; <em class='gold'>id- disa' mitt elf u mija u għaxra</em></div><div><b>NumAdjectival</b><em class='incorrect'>id- disat elf u mija u għaxar</em> &rarr; <em class='gold'>id- disa' mitt elf u mija u għaxar</em></div></div></div><hr/><div><b>s</b><div><b>NCard</b><div><b>NumNominative</b><em class='incorrect'>disat elf u mitejn</em> &rarr; <em class='gold'>disa' mitt elf u mitejn</em></div><div><b>NumAdjectival</b><em class='incorrect'>disat elf u mitejn</em> &rarr; <em class='gold'>disa' mitt elf u mitejn</em></div></div><div><b>NOrd</b><div><b>NumNominative</b><em class='incorrect'>id- disat elf u mitejn</em> &rarr; <em class='gold'>id- disa' mitt elf u mitejn</em></div><div><b>NumAdjectival</b><em class='incorrect'>id- disat elf u mitejn</em> &rarr; <em class='gold'>id- disa' mitt elf u mitejn</em></div></div></div><hr/>
</div>

</body>
</html>