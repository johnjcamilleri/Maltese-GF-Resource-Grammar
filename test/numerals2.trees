-- Run with:
-- rf -lines -tree -file=test/numerals2.trees | l -table

-- 1,000
num (pot3 (pot1as2 (pot0as1 pot01)))

-- 2,000
num (pot3 (pot1as2 (pot0as1 (pot0 n2))))

-- 3,000
num (pot3 (pot1as2 (pot0as1 (pot0 n3))))

-- 9,000
num (pot3 (pot1as2 (pot0as1 (pot0 n9))))

-- 10,000
num (pot3 (pot1as2 pot110))

-- 11,000
num (pot3 (pot1as2 pot111))

-- 12,000
num (pot3 (pot1as2 (pot1to19 n2)))

-- 14,000
num (pot3 (pot1as2 (pot1to19 n3)))

-- 19,000
num (pot3 (pot1as2 (pot1to19 n9)))

-- 20,000
num (pot3 (pot1as2 (pot1 n2)))

-- 21,000
num (pot3 (pot1as2 (pot1plus n2 pot01)))

-- 22,000
num (pot3 (pot1as2 (pot1plus n2 (pot0 n2))))

-- 30,000
num (pot3 (pot1as2 (pot1 n3)))

-- 31,000
num (pot3 (pot1as2 (pot1plus n3 pot01)))

-- 32,000
num (pot3 (pot1as2 (pot1plus n3 (pot0 n2))))

-- 99,000
num (pot3 (pot1as2 (pot1plus n9 (pot0 n9))))

-- 100,000
num (pot3 (pot2 pot01))

-- 101,000
num (pot3 (pot2plus pot01 (pot0as1 pot01)))

-- 106,000
num (pot3 (pot2plus pot01 (pot0as1 (pot0 n6))))

-- 110,000
num (pot3 (pot2plus pot01 pot110))

-- 111,000
num (pot3 (pot2plus pot01 pot111))

-- 130,000
num (pot3 (pot2plus pot01 (pot1 n3)))

-- 141,000
num (pot3 (pot2plus pot01 (pot1plus n4 pot01)))

-- 160,000
num (pot3 (pot2plus pot01 (pot1 n6)))

-- 200,000
num (pot3 (pot2 (pot0 n2)))

-- 416,000
num (pot3 (pot2plus (pot0 n4) (pot1to19 n6)))




-- 11,012
num (pot3plus (pot1as2 pot111) (pot1as2 (pot1to19 n2)))

-- 40,000
num (pot3 (pot1as2 (pot1 n4)))

-- 91,700
num (pot3plus (pot1as2 (pot1plus n9 pot01)) (pot2 (pot0 n7)))

-- 100,100
num (pot3plus (pot2 pot01) (pot2 pot01))

-- 100,031
num (pot3plus (pot2 pot01) (pot1as2 (pot1plus n3 pot01)))

-- 121,100
num (pot3plus (pot2plus pot01 (pot1plus n2 pot01)) (pot2 pot01))

-- 200,410
num (pot3plus (pot2 (pot0 n2)) (pot2plus (pot0 n4) pot110))

-- 600,000
num (pot3 (pot2 (pot0 n6)))

-- 610,010
num (pot3plus (pot2plus (pot0 n6) pot110) (pot1as2 pot110))

-- 700,610
num (pot3plus (pot2 (pot0 n7)) (pot2plus (pot0 n6) pot110))

-- 850,017
num (pot3plus (pot2plus (pot0 n8) (pot1 n5)) (pot1as2 (pot1to19 n7)))

-- 900,110
num (pot3plus (pot2 (pot0 n9)) (pot2plus pot01 pot110))

-- 900,200
num (pot3plus (pot2 (pot0 n9)) (pot2 (pot0 n2)))
