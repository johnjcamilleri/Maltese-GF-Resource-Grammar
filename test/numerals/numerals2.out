s NCard NumNominative : elf
s NCard NumAdjectival : elf
s NOrd NumNominative : l- &+ elf
s NOrd NumAdjectival : l- &+ elf


s NCard NumNominative : elfejn
s NCard NumAdjectival : elfejn
s NOrd NumNominative : l- &+ elfejn
s NOrd NumAdjectival : l- &+ elfejn


s NCard NumNominative : tlett elef
s NCard NumAdjectival : tlett elef
s NOrd NumNominative : it- &+ tlett elef
s NOrd NumAdjectival : it- &+ tlett elef


s NCard NumNominative : disat elef
s NCard NumAdjectival : disat elef
s NOrd NumNominative : id- &+ disat elef
s NOrd NumAdjectival : id- &+ disat elef


s NCard NumNominative : għaxart elef
s NCard NumAdjectival : għaxart elef
s NOrd NumNominative : l- &+ għaxart elef
s NOrd NumAdjectival : l- &+ għaxart elef


s NCard NumNominative : ħdax-il elf
s NCard NumAdjectival : ħdax-il elf
s NOrd NumNominative : il- &+ ħdax-il elf
s NOrd NumAdjectival : il- &+ ħdax-il elf


s NCard NumNominative : tnax-il elf
s NCard NumAdjectival : tnax-il elf
s NOrd NumNominative : it- &+ tnax-il elf
s NOrd NumAdjectival : it- &+ tnax-il elf


s NCard NumNominative : tlettax-il elf
s NCard NumAdjectival : tlettax-il elf
s NOrd NumNominative : it- &+ tlettax-il elf
s NOrd NumAdjectival : it- &+ tlettax-il elf


s NCard NumNominative : dsatax-il elf
s NCard NumAdjectival : dsatax-il elf
s NOrd NumNominative : id- &+ dsatax-il elf
s NOrd NumAdjectival : id- &+ dsatax-il elf


s NCard NumNominative : għoxrin elf
s NCard NumAdjectival : għoxrin elf
s NOrd NumNominative : l- &+ għoxrin elf
s NOrd NumAdjectival : l- &+ għoxrin elf


s NCard NumNominative : wieħed u għoxrin elf
s NCard NumAdjectival : wieħed u għoxrin elf
s NOrd NumNominative : il- &+ wieħed u għoxrin elf
s NOrd NumAdjectival : il- &+ wieħed u għoxrin elf


s NCard NumNominative : tnejn u għoxrin elf
s NCard NumAdjectival : tnejn u għoxrin elf
s NOrd NumNominative : it- &+ tnejn u għoxrin elf
s NOrd NumAdjectival : it- &+ tnejn u għoxrin elf


s NCard NumNominative : tletin elf
s NCard NumAdjectival : tletin elf
s NOrd NumNominative : it- &+ tletin elf
s NOrd NumAdjectival : it- &+ tletin elf


s NCard NumNominative : wieħed u tletin elf
s NCard NumAdjectival : wieħed u tletin elf
s NOrd NumNominative : il- &+ wieħed u tletin elf
s NOrd NumAdjectival : il- &+ wieħed u tletin elf


s NCard NumNominative : tnejn u tletin elf
s NCard NumAdjectival : tnejn u tletin elf
s NOrd NumNominative : it- &+ tnejn u tletin elf
s NOrd NumAdjectival : it- &+ tnejn u tletin elf


s NCard NumNominative : disgħa u disgħin elf
s NCard NumAdjectival : disgħa u disgħin elf
s NOrd NumNominative : id- &+ disgħa u disgħin elf
s NOrd NumAdjectival : id- &+ disgħa u disgħin elf


s NCard NumNominative : mitt elf
s NCard NumAdjectival : mitt elf
s NOrd NumNominative : il- &+ mitt elf
s NOrd NumAdjectival : il- &+ mitt elf


s NCard NumNominative : mija u wieħed elf
s NCard NumAdjectival : mija u wieħed elf
s NOrd NumNominative : il- &+ mija u wieħed elf
s NOrd NumAdjectival : il- &+ mija u wieħed elf


s NCard NumNominative : mija u sitt elef
s NCard NumAdjectival : mija u sitt elef
s NOrd NumNominative : il- &+ mija u sitt elef
s NOrd NumAdjectival : il- &+ mija u sitt elef


s NCard NumNominative : mija u għaxart elef
s NCard NumAdjectival : mija u għaxart elef
s NOrd NumNominative : il- &+ mija u għaxart elef
s NOrd NumAdjectival : il- &+ mija u għaxart elef


s NCard NumNominative : mija u ħdax-il elf
s NCard NumAdjectival : mija u ħdax-il elf
s NOrd NumNominative : il- &+ mija u ħdax-il elf
s NOrd NumAdjectival : il- &+ mija u ħdax-il elf


s NCard NumNominative : mija u tletin elf
s NCard NumAdjectival : mija u tletin elf
s NOrd NumNominative : il- &+ mija u tletin elf
s NOrd NumAdjectival : il- &+ mija u tletin elf


s NCard NumNominative : mija u wieħed u erbgħin elf
s NCard NumAdjectival : mija u wieħed u erbgħin elf
s NOrd NumNominative : il- &+ mija u wieħed u erbgħin elf
s NOrd NumAdjectival : il- &+ mija u wieħed u erbgħin elf


s NCard NumNominative : mija u sittin elf
s NCard NumAdjectival : mija u sittin elf
s NOrd NumNominative : il- &+ mija u sittin elf
s NOrd NumAdjectival : il- &+ mija u sittin elf


s NCard NumNominative : mitejn elf
s NCard NumAdjectival : mitejn elf
s NOrd NumNominative : il- &+ mitejn elf
s NOrd NumAdjectival : il- &+ mitejn elf


s NCard NumNominative : erba' mija u sittax-il elf
s NCard NumAdjectival : erba' mija u sittax-il elf
s NOrd NumNominative : l- &+ erba' mija u sittax-il elf
s NOrd NumAdjectival : l- &+ erba' mija u sittax-il elf


s NCard NumNominative : ħdax-il elf u tnax
s NCard NumAdjectival : ħdax-il elf u tnax-il
s NOrd NumNominative : il- &+ ħdax-il elf u tnax
s NOrd NumAdjectival : il- &+ ħdax-il elf u tnax-il


s NCard NumNominative : erbgħin elf
s NCard NumAdjectival : erbgħin elf
s NOrd NumNominative : l- &+ erbgħin elf
s NOrd NumAdjectival : l- &+ erbgħin elf


s NCard NumNominative : wieħed u disgħin elf u seba' mija
s NCard NumAdjectival : wieħed u disgħin elf u seba' mitt
s NOrd NumNominative : il- &+ wieħed u disgħin elf u seba' mija
s NOrd NumAdjectival : il- &+ wieħed u disgħin elf u seba' mitt


s NCard NumNominative : mitt elf u mija
s NCard NumAdjectival : mitt elf u mitt
s NOrd NumNominative : il- &+ mitt elf u mija
s NOrd NumAdjectival : il- &+ mitt elf u mitt


s NCard NumNominative : mitt elf u wieħed u tletin
s NCard NumAdjectival : mitt elf u wieħed u tletin
s NOrd NumNominative : il- &+ mitt elf u wieħed u tletin
s NOrd NumAdjectival : il- &+ mitt elf u wieħed u tletin


s NCard NumNominative : mija u wieħed u għoxrin elf u mija
s NCard NumAdjectival : mija u wieħed u għoxrin elf u mitt
s NOrd NumNominative : il- &+ mija u wieħed u għoxrin elf u mija
s NOrd NumAdjectival : il- &+ mija u wieħed u għoxrin elf u mitt


s NCard NumNominative : mitejn elf u erba' mija u għaxra
s NCard NumAdjectival : mitejn elf u erba' mija u għaxar
s NOrd NumNominative : il- &+ mitejn elf u erba' mija u għaxra
s NOrd NumAdjectival : il- &+ mitejn elf u erba' mija u għaxar


s NCard NumNominative : sitt elf
s NCard NumAdjectival : sitt elf
s NOrd NumNominative : is- &+ sitt elf
s NOrd NumAdjectival : is- &+ sitt elf


s NCard NumNominative : sitt mija u għaxart elef u għaxra
s NCard NumAdjectival : sitt mija u għaxart elef u għaxar
s NOrd NumNominative : is- &+ sitt mija u għaxart elef u għaxra
s NOrd NumAdjectival : is- &+ sitt mija u għaxart elef u għaxar


s NCard NumNominative : sebat elf u sitt mija u għaxra
s NCard NumAdjectival : sebat elf u sitt mija u għaxar
s NOrd NumNominative : is- &+ sebat elf u sitt mija u għaxra
s NOrd NumAdjectival : is- &+ sebat elf u sitt mija u għaxar


s NCard NumNominative : tmin mija u ħamsin elf u sbatax
s NCard NumAdjectival : tmin mija u ħamsin elf u sbatax-il
s NOrd NumNominative : it- &+ tmin mija u ħamsin elf u sbatax
s NOrd NumAdjectival : it- &+ tmin mija u ħamsin elf u sbatax-il


s NCard NumNominative : disat elf u mija u għaxra
s NCard NumAdjectival : disat elf u mija u għaxar
s NOrd NumNominative : id- &+ disat elf u mija u għaxra
s NOrd NumAdjectival : id- &+ disat elf u mija u għaxar


s NCard NumNominative : disat elf u mitejn
s NCard NumAdjectival : disat elf u mitejn
s NOrd NumNominative : id- &+ disat elf u mitejn
s NOrd NumAdjectival : id- &+ disat elf u mitejn



80 msec
AllMltAbs> See you.
0 msec